crates/symbolicator-js/src/symbolication.rs

```diff
@@ -307,7 +316,7 @@ async fn symbolicate_js_frame(
 }
 
 fn apply_source_context(frame: &mut JsFrame, source: &str) -> Result<(), JsModuleErrorKind> {
-    let lineno = frame.lineno as usize;
+    let lineno = frame.lineno.map(|line| line as usize).unwrap_or_default();
```

Comment:
If the line is `None` I don't think we should even attempt to lookup a source context and just exit here with `Ok(())`